docs/contributing.md

```diff
@@ -53,32 +52,38 @@ If you are proposing a feature:
 
 Ready to contribute? Here's how to set up `python-statemachine` for local development.
 
+1. Install dependencies.
```

Comment:
**thought:** If you install poetry. After running `poetry install`, run `poetry shell` to activate the virtualenv on the shell. So `ruff` and `pre-commit` will be available and we don't need to install them globally.

So the external pre-conditions are only poetry and graphviz.

Reply:
Well at least for me it was not working.

I get e.g. this error when I do not install graphviz - globally:

    FAILED tests/test_contrib_diagram.py::test_machine_repr_custom_[expected_reprs1-AllActionsMachine] - FileNotFoundError: [Errno 2] "dot" not found in path

Reply:
Poetry and graphviz are the only required pre-conditions, right? I was referring to the ruff and pre-commit that may be used from the virtualenv created by poetry.